app/main.js

```diff
@@ -20,3 +20,11 @@ if (process.env.NODE_ENV !== 'production' || process.env.HAS_STYLEGUIDE) {
 }
 
 window.vm = new Vue(vueOptions);
+
+// Set's current theme id to the vuex store read by url param for e.g. ?theme=01
+const themeId = window.location.search.split('theme=')[1];
```

Comment:
- [ ] Setting theme by query does have no impact on the selected entry in the theme-selector. Please check Vuex binding.